requirements/dev.in

```diff
@@ -26,7 +26,5 @@ seqdiag
 sphinx-automodapi
 sphinx-click
 sphinx-prompt
-
-# Temporary to prevent conflicts with reuqirements/main.txt due to a different
-# click requirement from uvicorn.
-uvicorn
+types-cachetools
```

Comment:
Did mypy just roll out a new feature requiring stubs? I just noticed this too.

Reply:
Yeah, it looks like they took all the third-party stubs out of typeshed and split them into separate packages. I suspect they're trying to get rid of typeshed and push the type information down into the standard library and the source packages or, failing that, separate PyPI packages instead of maintaining them all as part of mypy (for understandable reasons).